lib/entities/editor-interface.ts

```diff
@@ -72,6 +93,10 @@ export type EditorInterfaceProps = {
    * Array of fields and it's associated widgetId
    */
   controls?: Control[]
+  /**
+   * Array of groups and it's associated widgetId
```

Comment:
Probably cloned from above
```suggestion
   * Array of groups and their associated widgetId
```

Reply:
fixed